promptsource/templates/xnli/hi/templates.yaml

```diff
@@ -1,6 +1,31 @@
 dataset: xnli
 subset: hi
 templates:
+  25a32b2a-f5e2-4e74-a5c9-828b99261b43: !Template
+    answer_choices: सही ||| दुविधा में पड़ा हुआ ||| ग़लत
```

Comment:
```suggestion
    answer_choices: सही ||| निश्चित रूप से सही ||| ग़लत
```

Reply:
This prompt is machine translated (mt), so it's okay if it has mistakes!
We're interested in comparing whether ht <-> mt makes a difference

Reply:
Oh, sorry! My bad.